tests/ManageCourses.Tests/SmokeTests/TestPayloadBuilder.cs

```diff
@@ -10,7 +11,8 @@ public static class TestPayloadBuilder {
                 Users = ListOfOne(new McUser{
                     FirstName = "Joe",
                     LastName = "Bloggs",
-                    Email = username
+                    Email = username,
+                    AcceptTermsDateUtc = DateTime.Now
```

Comment:
Should use NowUtc

reject

p.s. should probably also be using the mocked times to avoid timing based test failures. there's already a mock clock etc in the base class

Reply:
oops (UtcNow)